corehq/apps/analytics/tasks.py

```diff
@@ -76,7 +76,8 @@ def _link_account_with_cookie(webuser, cookies, meta):
             'email': webuser.username,
             'firstname': webuser.first_name,
             'lastname': webuser.last_name,
-            'hs_context': json.dumps({"hutk": hubspot_cookie, "ipAddress": _get_client_ip(meta)})
+            'hs_context': json.dumps({"hutk": hubspot_cookie, "ipAddress": _get_client_ip(meta)}),
+            'opt_into_emails': meta['opt_into_emails'],
```

Comment:
is there ever a chance of this key not being in the dict? if so might be good to use `.get` with a default, but if it's always set then this is better